src/createContext.spec.lua

```diff
@@ -0,0 +1,168 @@
+return function()
+	local createContext = require(script.Parent.createContext)
+	local createElement = require(script.Parent.createElement)
+	local NoopRenderer = require(script.Parent.NoopRenderer)
+	local createReconciler = require(script.Parent.createReconciler)
+	local PureComponent = require(script.Parent.PureComponent)
+
+	local noopReconciler = createReconciler(NoopRenderer)
+
+	it("should return a table", function()
+		local context = createContext("Test")
+		expect(context).to.be.ok()
+		expect(type(context)).to.equal("table")
+	end)
+
+	it("should contain a Provider and a Consumer", function()
+		local context = createContext("Test")
+		expect(context.Provider).to.be.ok()
+		expect(context.Consumer).to.be.ok()
+	end)
+
+	describe("Provider", function()
+		it("should render its child component", function()
+			local didRender = false
+			local context = createContext("Test")
+
+			local Listener = PureComponent:extend("Listener")
```

Comment:
Instead of a `PureComponent`, this should just be a function component for simplicity.